src/core/application.cpp

```diff
@@ -66,20 +84,55 @@ void Application::waitBootingBmc()
     // query to dbus-services which an incomplete state.
     while(std::filesystem::exists(bootingFilePath))
     {
+        log<level::INFO>("Wait for BMC boot completed...");
         std::this_thread::sleep_for(1s);
     }
+
+    log<level::INFO>("The BMC boot state is OK");
```

Comment:
Are you sure we need to log this at `INFO` level? Won't this be filling the logs with the information that has little use to the end user?